ext/bg/js/search-query-parser-generator.js

```diff
@@ -71,7 +71,16 @@ class QueryParserGenerator {
         for (const parseResult of parseResults) {
             const optionContainer = this._templateHandler.instantiate('select-option');
             optionContainer.value = parseResult.id;
-            optionContainer.textContent = parseResult.name;
+            switch (parseResult.source) {
+                case 'scanning-parser':
+                    optionContainer.textContent = 'Scanning parser';
+                    break;
+                case 'mecab':
+                    optionContainer.textContent = `MeCab: ${parseResult.dictionary}`;
+                    break;
+                default:
+                    optionContainer.textContent = 'Unrecognized dictionary';
```

Comment:
break

Reply:
(I'm used to some other languages requiring it; unfortunately [not enforceable](https://github.com/eslint/eslint/issues/7697).)

Reply:
Yeah, I think it's good practice as well although not required by the language. I actually like trailing commas as well for the same reason, but we've opted to not use them.

Reply:
https://github.com/FooSoft/yomichan/pull/456/commits/a02cf28c5775bdd72f209b46e9974768e5c5db71